docs/.vitepress/theme/index.ts

```diff
@@ -1,19 +1,31 @@
 import ElementPlus from 'element-plus'
+import { createPinia } from 'pinia'
 
 import VPApp, { NotFound, globals } from '../vitepress'
 import { define } from '../utils/types'
 import 'uno.css'
 import './style.css'
 import type { Theme } from 'vitepress'
 
+import {} from '../'
```

Comment:
What is the purpose of writing this way? 🤔

Reply:
It's a mistake. I will remove it.